rules-tests/TypeDeclaration/Rector/Property/TypedPropertyFromAssignsRector/Fixture/nullable_if_checked.php.inc

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\TypedPropertyFromAssignsRector\Fixture;
+
+use Rector\Tests\TypeDeclaration\Rector\Property\TypedPropertyFromAssignsRector\Source\SomeConnection;
+
+final class NullableIfChecked
+{
+    /**
+     * @var SomeConnection
+     */
+    private $connection;
+
+    public function __construct()
+    {
+        if ($this->connection == null) {
+            $this->initConnection();
+        }
+        return $this->connection;
```

Comment:
`return` with expression in `__construct()` is invalid btw, we have `RemoveUselessReturnExprInConstructRector` for that 

https://getrector.com/demo/b3f08404-ae65-4447-942b-38ace79c2af3

Reply:
https://github.com/rectorphp/rector-src/pull/6101